opentech/apply/dashboard/tables.py

```diff
@@ -67,3 +70,34 @@ class SubmissionFilter(filters.FilterSet):
     class Meta:
         model = ApplicationSubmission
         fields = ('funds', 'round', 'status')
+
+
+class JSONSearchFilter(filters.CharFilter):
```

Comment:
Given you store the submission as text, is the JSONFilter needed?
What about using an [HStoreField](https://docs.djangoproject.com/en/2.0/ref/contrib/postgres/fields/#django.contrib.postgres.fields.HStoreField) which has the benefit of a [values()](https://docs.djangoproject.com/en/2.0/ref/contrib/postgres/fields/#values) lookup ?

Reply:
Been considering it. Most of the data is going to be in a single hierarchy. There is a small element of nesting but i don't think we will really need to query over it.